develop/parachain-devs/runtime-development/FRAME/add-pallet-to-the-runtime.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Add Pallet to the Runtime
+description: Learn how to include and configure pallets in a Polkadot SDK-based runtime, from adding dependencies to implementing necessary traits.
+---
+
+# Add Pallet to the Runtime
+
+## Introduction
+
+As demonstrated in the [Build a Local Solochain](/tutorials/polkadot-sdk/build-a-blockchain/build-a-local-blockchain/){target=\_blank} guide, the [Polkadot SDK Solochain Template](https://github.com/paritytech/polkadot-sdk-solochain-template){target=\_blank} provides a functional runtime that includes default FRAME development modules—pallets—to help you get started with building a custom blockchain.
+
+This article outlines the basic steps for adding a new pallet to the runtime of the node template. These steps apply when integrating a FRAME pallet into your runtime. However, each pallet has specific configuration requirements, such as the parameters and types needed to enable the pallet's functionality. In this guide, you'll learn how to add a pallet to the runtime and configure the settings specific to that pallet. For instance, some pallets may allow users to interact with the blockchain by setting or modifying data, while others may manage transactions or permissions.
```

Comment:
> "For instance, some pallets may allow users to interact with the blockchain by setting or modifying data, while others may manage transactions or permissions."

I'm having trouble understanding this. Could you let me know some pallet examples for both of these scenarios? 

Perhaps, this can be rephrased to keep the discussion around configuring the settings of the pallet, as it starts with "for instance".

Reply:
yes, I re-read this one and it's really hard to understand tbh, let me tweak it a bit